tests/integration/security/chiron/dns_cert_test.go

```diff
@@ -93,6 +93,7 @@ ksOPXgK63Oot7wxQOuG5BX1v1yQ=
 
 func TestDNSCertificate(t *testing.T) {
 	framework.NewTest(t).
+		Features("security.control-plane.k8s-certs").
```

Comment:
Since we are using this feature in multiple tests, it might be useful to track which scenario the test is targeting.  Note that scenarios don't need to appear in features.yaml.

```suggestion
		Features("security.control-plane.k8s-certs.dns-certificate").
```
(this is just my uneducated guess as to what scenario you are testing here)

Reply:
Could you clarify it more? 
I am not very suire what is the scenario for dns-certificate . is it a similar feature like the other label(e.g. k8s-certs, ingress.tls) Could you help me clarify it? Thank you very much~~